packages/sampler/.storybook/preview.js

```diff
@@ -1,6 +1,6 @@
 import {configureActions} from '@enact/storybook-utils/addons/actions';
 import {getBooleanType, getObjectType} from '@enact/storybook-utils/addons/controls';
-import {DocsContainer, Primary, Title} from '@enact/storybook-utils/addons/docs';
+import {DocsContainer, Primary, Title} from '@storybook/addon-docs';
```

Comment:
One last question.. Do we need to use this? I don't see any docs here.

Reply:
no, we don't at the moment. I removed te entire docs configuration